[ACR-1811] Deprecate underutilized fields (#1919)
Samsung/tizen-docs: docs/application/native/guides/multimedia/media-content.md

                   free(datetaken);

               image_meta_destroy(image_handle);
           } else if (media_type == MEDIA_CONTENT_TYPE_MUSIC) {
               audio_meta_h audio_handle;
               char *title = NULL;
               char *artist = NULL;
               char *album = NULL;
               char *album_artist = NULL;

               ret = media_info_get_audio(media_handle, &audio_handle);
               if (ret != MEDIA_CONTENT_ERROR_NONE) {
                   /* Error handling */
               } else {
                   audio_meta_get_artist(audio_handle, &artist);
                   audio_meta_get_album(audio_handle, &album);
                   audio_meta_get_album_artist(audio_handle, &album_artist);

>                    dlog_print(DLOG_DEBUG, LOG_TAG, "This is a music");

Reviewer: ```suggestion
                   dlog_print(DLOG_DEBUG, LOG_TAG, "This is an audio file");
```
